test/verify/check-setroubleshoot

```diff
@@ -134,17 +134,8 @@ class TestSelinux(MachineCase):
         btn_sel = "div.list-group-item:contains('you can run restorecon') button"
         b.click(btn_sel)
 
-        # HACK there's a known issue where setroubleshoot is unable to run the fix
         # the fix should be applied
         b.wait_in_text(row_selector, "Solution applied successfully:")
-        # b.wait_in_text(row_selector, "Successfully ran /sbin/restorecon")
-
-        # HACK: we need updated test images for delete to work
-        # dismiss the alert
-        #b.click(row_selector + " button.pficon-delete")
-        #b.wait_not_present(row_selector)
-
-        #b.wait_in_text("body", "No SELinux alerts.")
```

Comment:
So dismissing an alert still doesn't work? That seems rather unrelated to enabling audit to spot them in the first place.

I realize that this code is already commented out in master, but would this work again?

Reply:
I tested that on Fedora-26. Lines 140, 144, and 145 do work again, but the very last line 147 doesn't, as apparently there is a second alert still:

![testselinux-testtroubleshootalerts-fedora-26-127 0 0 2-2201-fail](https://user-images.githubusercontent.com/200109/30366687-3fb2fc14-986c-11e7-841b-13522372c073.png)

I think that's moderately legitimate, so let's drop line 147 for good, and re-enable the others?

Reply:
I'm not sure they work on rhel, but i'll push a change to check